docs/setup/settings.asciidoc

```diff
@@ -114,30 +114,42 @@ in this setting.
 
 `elasticsearch.logQueries`::
 deprecated:[7.12.0,"This setting is no longer used and will be removed in Kibana 8.0."]
-Instead, configure the `elasticsearch.query` logger.
+Instead, configure the `elasticsearch.query` logger and an appender for it.
 +
 This is useful for seeing the query DSL generated by applications that
 currently do not have an inspector, for example Timelion and Monitoring.
 *Default: `false`*
 +
-The following example shows a valid `elasticsearch.query` logger configuration:
+The following example shows a full, valid `elasticsearch.query` logging configuration:
 +
-[source,text]
---
+[source,yaml]
+--------------------------------------------------------------------------------
 logging:
   appenders:
-    console_appender:
-      type: console
+    console_appender: <1>
+      type: console <2>
       layout:
-        type: pattern
+        type: pattern <3>
         highlight: true
   root:
     appenders: [default, console_appender]
     level: warn
   loggers:
-    - name: elasticsearch.query
+    - name: elasticsearch.query <4>
       level: debug
---
+      appenders: [console_appender]
```

Comment:
nit: technically this isn't necessary for the example to run, but I guess it shows how you can change appenders per logger

Reply:
Adding an example is part of the "soft introduction" to the new logging system. It also shows how to specify an appender for a specific logger context and not fall back to the root logger config.